pkg/api/exp/oneagent.go

```diff
@@ -0,0 +1,68 @@
+package exp
+
+const (
+	// Deprecated: OAProxyIgnoredKey use NoProxy annotation instead.
+	OAProxyIgnoredKey = FFPrefix + "oneagent-ignore-proxy"
+
+	OAMaxUnavailableKey      = FFPrefix + "oneagent-max-unavailable"
+	OAInitialConnectRetryKey = FFPrefix + "oneagent-initial-connect-retry-ms"
+	OAPrivilegedKey          = FFPrefix + "oneagent-privileged"
+	OASkipLivenessProbeKey   = FFPrefix + "oneagent-skip-liveness-probe"
+
+	// Deprecated: Dedicated field since v1beta2.
+	OASecCompProfileKey = FFPrefix + "oneagent-seccomp-profile"
+
+	OANodeImagePullKey = FFPrefix + "node-image-pull"
+	// OANodeImagePullTechnologiesKey can be set on a Pod or DynaKube to configure which code module technologies to download. It's set to
+	// "all" if not set.
+	OANodeImagePullTechnologiesKey = "oneagent.dynatrace.com/technologies"
```

Comment:
I don't know if the prefix `"oneagent.dynatrace.com"` will be used more often in the future, but if so, it would make sense to create a const for it like you did for `FFPrefix`

Reply:
kk.